packages/edit-site/src/components/template-switcher/index.js

```diff
@@ -2,8 +2,9 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { useSelect } from '@wordpress/data';
-import { useState } from '@wordpress/element';
+import { addQueryArgs } from '@wordpress/url';
+import { resolveSelect, useDispatch, useSelect } from '@wordpress/data';
```

Comment:
I think `resolveSelect` is still experimental.

Reply:
Ohh indeed :man_facepalming:

Reply:
Fixed by 59d0688244fd4b51ead82c889faf07e68c992db6